std/encoding.d

```diff
@@ -2233,7 +2101,7 @@ body
     }
     else static if(is(Src==AsciiChar))
     {
-        transcode!(char,Dst)(cast(string)s,r);
+        transcodeFromAscii(s, r);
```

Comment:
Please instead do:
```d
transcode!(char, Dst)(() @trusted { return cast(string)s; } (), r); // @@@BUG@@@ 15762
```
With a private function at module scope it's not clear to what extent the function is used. I only understood it when seeing this line.